CorsixTH/Lua/dialogs/resizables/new_game.lua

```diff
@@ -196,17 +196,18 @@ function UINewGame:buttonStart()
 end
 
 function UINewGame:startGame(difficulty)
-  if self.ui.app:loadLevel(1, difficulty, nil, nil, nil, nil,
-      _S.errors.load_level_prefix, nil) then
-    self.ui.app.moviePlayer:playAdvanceMovie(1)
+  local app = self.ui.app
```

Comment:
Why did you add this? `TheApp` is a defined global name for `app`.

I know we have code that pulls the value from many random spots, but there really isn't need for that.